twitter-covid/period-most-discussed/src/main/scala/Main.scala

```diff
@@ -0,0 +1,14 @@
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import peakDiscussion._
+object Main {
+    def main(args: Array[String]): Unit = {
+        val spark = SparkSession.builder()
+        .appName("peakDiscussion")
+        .master("yarn") //Switch to "yarn" for running on EMR, "*" for local.
+        .getOrCreate()
+        spark.sparkContext.setLogLevel("WARN")
+        configureAWS(spark) //Only necessary when running locally.
```

Comment:
if line 7 is yarn then should this line be commented or does the comment on this line need to change.